### PR TITLE
Feature/onboarding

### PR DESCRIPTION
**Pull request by rahulmadishetty**

### CODE REVIEW

File: frontend/src/pages/Onboarding/mainonboarding.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function mainonboarding() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/mainonboarding.jsx:3]: Lets rename the file to MainOnboarding. Normally we use PascalCase naming convention for react components. Lets also use Arrow functions like this:
```
const MainOnbaording = () => {

return <>...</>

}
```

File: frontend/src/pages/Onboarding/multistepprogressbar.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function multistepprogressbar() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/multistepprogressbar.jsx:3]: Same here.

File: frontend/src/pages/Onboarding/pagefour.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function pagefour() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/pagefour.jsx:3]: Same here.

File: frontend/src/pages/Onboarding/pageone.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function pageone() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/pageone.jsx:3]: Lets rename the page to some specific term for the components which will be easier to understand. For example HomePage, Dashboard etc.

File: frontend/src/pages/Onboarding/pagethree.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function pagethree() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/pagethree.jsx:3]: Same here.

File: frontend/src/pages/Onboarding/pagetwo.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function pagetwo() {
```
Comment (anishapant21) [inline frontend/src/pages/Onboarding/pagetwo.jsx:3]: Same here.

File: frontend/src/pages/Onboarding/multistepprogressbar.jsx (hunk 2 above)
Comment (rahulmadishetty) [reply frontend/src/pages/Onboarding/multistepprogressbar.jsx]: resolved here as well

File: frontend/src/pages/Onboarding/mainonboarding.jsx (hunk 1 above)
Comment (rahulmadishetty) [reply frontend/src/pages/Onboarding/mainonboarding.jsx]: Resolved

**Pull request merged**